core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.html

```diff
@@ -19,27 +19,30 @@
             </ng-template>
         </p-tabPanel>
         <p-tabPanel
-            *ngFor="let containerContent of containerContents.controls; let i = index"
-            [header]="containerContent.value.name"
+            *ngFor="let containerContent of getcontainerStructures.controls; let i = index"
+            [header]="getContentTypeNameById(containerContent.value.structureId)"
```

Comment:
https://plainenglish.io/blog/angular-dont-call-a-function-inside-the-template-anymore